editoast/src/main.rs

```diff
@@ -33,7 +33,6 @@ use client::user::UserCommand;
 use editoast_common::tracing::TracingConfig;
 use editoast_common::tracing::create_tracing_subscriber;
 use editoast_models::DbConnectionPoolV2;
-use models::RollingStockModel;
```

Comment:
I'm curious, why this change happens... and why is there no compilation error when you remove it?

Reply:
To be honest, I stumbled across it without understanding what it was doing there, and when I removed it... well, nothing happened.